code/drug_synergy/utils/get_synergy_score.R

```diff
@@ -0,0 +1,37 @@
+
+
+suppressPackageStartupMessages(library("optparse"))
+suppressPackageStartupMessages(library("tidyverse"))
+suppressPackageStartupMessages(library("readr"))
+suppressPackageStartupMessages(library("igraph"))
+
+#### Define Directories ----------------------------------------------------------
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+analysis_dir <- file.path(root_dir, "code", "drug_synergy")
+results_dir <- file.path(analysis_dir, "results", "synergy_score")
+if(!dir.exists(results_dir)){
+  dir.create(results_dir, recursive = TRUE)
+}
+getSynScore2 <- function(tar1, tar2, sigNet1){
```

Comment:
@runjin326 - can you add a Roxygen skeleton here that defines the function's purpose and parameter definitions?

Reply:
Added! Please let me know whether it's clear enough.